### DISCUSSION TITLE
Subsequent tasks, or one task depending on other (dependsOn equivalent on VSCode)

### DISCUSSION THREAD

**Discussion by dimitre** [zed-industries/zed#22201] [Feature Requests]
I'm trying to use ZED as an editor / IDE for openFrameworks
and it would be great if we could have one task that invokes two others, like
Build and Run
but if build fails, run is not invoked.

on VSCode it is possible something like this:
```
		{
			"label": "Build and Run Release",
			"dependsOn": ["Build RELEASE", "Run RELEASE"],
			"dependsOrder": "sequence",
			"group": {
				"kind": "build",
				"isDefault": true
			}
		},
```
it would be great to be able to do the same using ZED.
Congratulations on the project, it works great, looks good and development is fast!

**Comment by bobbymannino** [comment]
this would make a fantastic addition to the "IDE" experience, hopefully this gets implimented

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
